src/babel/api/register/node.js

```diff
@@ -76,6 +78,9 @@ var compile = function (filename) {
 };
 
 var shouldIgnore = function (filename) {
+  if (isWindows) {
+    filename = filename.split("\\").join("/");
```

Comment:
There's the [`slash`](https://github.com/sindresorhus/slash) package that normalises these so this is unnecessary. You can just do `import slash from "slash"; filename = slash(filename);`